app/jobs/resolution_proofing_job.rb

```diff
@@ -131,71 +131,57 @@ def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
 
   # @return [CallbackLogData]
   def proof_lexisnexis_then_aamva(timer:, applicant_pii:, should_proof_state_id:)
-    proofer_result = timer.time('resolution') do
+    resolution_result = timer.time('resolution') do
       resolution_proofer.proof(applicant_pii)
     end
 
-    result = proofer_result.to_h
-    resolution_success = proofer_result.success?
-
-    result[:transaction_id] = proofer_result.transaction_id
-    result[:reference] = proofer_result.reference
-
-    exception = proofer_result.exception.inspect if proofer_result.exception
-    result[:timed_out] = proofer_result.timed_out?
-    result[:exception] = exception
+    state_id_result = Proofing::Aamva::UnsupportedJurisdictionResult.new
+    if should_proof_state_id && resolution_result.success?
+      timer.time('state_id') do
+        state_id_result = state_id_proofer.proof(applicant_pii)
+      end
+    end
 
-    result[:context] = {
-      should_proof_state_id: should_proof_state_id,
-      stages: {
-        resolution: {
-          client: resolution_proofer.class.vendor_name,
-          errors: proofer_result.errors,
-          exception: exception,
-          success: proofer_result.success?,
-          timed_out: proofer_result.timed_out?,
-          transaction_id: proofer_result.transaction_id,
-          reference: proofer_result.reference,
+    result = {
+      success: resolution_result.success?,
+      errors: resolution_result.errors.merge(state_id_result.errors),
+      exception: resolution_result.exception || state_id_result.exception,
+      timed_out: resolution_result.timed_out? || state_id_result.timed_out?,
+      context: {
+        should_proof_state_id: should_proof_state_id,
+        stages: {
+          resolution: {
+            client: resolution_proofer.class.to_s,
+            errors: resolution_result.errors,
+            exception: resolution_result.exception,
+            success: resolution_result.success?,
+            timed_out: resolution_result.timed_out?,
+            transaction_id: resolution_result.transaction_id,
+            reference: resolution_result.reference,
+            # rubocop:disable Layout/LineLength
+            can_pass_with_additional_verification: resolution_result.failed_result_can_pass_with_additional_verification?,
+            attributes_requiring_additional_verification: resolution_result.attributes_requiring_additional_verification,
+            # rubocop:enable Layout/LineLength
+          },
+          state_id: {
```

Comment:
It looks like [this code](https://github.com/18F/identity-idp/blob/fc4987b70018434a7dc8daf2adcb5da6f5913d39/app/services/idv/steps/verify_base_step.rb#L47) depended on the absence of a proofing state_id stage to not add AAMVA proofing costs and components and all that magic. We may need to change that up to look at the client/vendor name here and be a little smarter.